### PR TITLE
Branch login admin

### PR DESCRIPTION
**Pull request by ghost**
@at-vienlm Edit  view login admin. Please help me check. thanks

### CODE REVIEW

File: resources/views/admin/layouts/partials/_navbar.blade.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <a class="navbar-brand" href="index.html">ADMIN</a>
+                <a class="navbar-brand" href="index.html"><span style="color:green;">Borrowing Book Admin</span></a>
```
Comment (dzungnguyenh) [inline resources/views/admin/layouts/partials/_navbar.blade.php]: [rank A] We need define all labels or texts in language files. How do you think?

File: resources/views/admin/layouts/partials/_navbar.blade.php [hunk 2]
```diff
@@ -92,7 +92,7 @@
                     </ul>
                 </li>
                 <li class="dropdown">
-                    <a href="#" class="dropdown-toggle" data-toggle="dropdown"><i class="fa fa-user"></i> {{ Auth::guard('admin')->user()->username }}<b class="caret"></b></a>
+                    <a href="#" class="dropdown-toggle" data-toggle="dropdown"><i class="fa fa-user"></i><span style="color: #1abc9c;"> {{ Auth::guard('admin')->user()->username }}</span><b class="caret"></b></a>
```
Comment (dzungnguyenh) [inline resources/views/admin/layouts/partials/_navbar.blade.php]: [rank B] please avoid to use css or js inline.

**Comment by ghost** [comment]
@at-dungnh i fix css for text or label. Please check again, thank you!

**Comment by ghost** [comment]
@at-dungnh i fixed

**Comment by dzungnguyenh** [comment]
Perfect! Thank you.

**Pull request merged**